docs/privacy-policy.md

```diff
@@ -0,0 +1,78 @@
+# Privacy Policy
+
+#### _Effective:_ February 8, 2022
+
+
+This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.
+
+1. [Collection Of Information](#Collection-Of-Information)
+
+2. [Use Of Information](#Use-Of-Information)
+
+3. [Sharing Of Information](#Sharing-Of-Information)
+
+4. [Additional Details](#Additional-Details)
+
+5. [References](#References)
+
+6. [Contact Us](#Contact-Us)
+
+
+
+## Collection Of Information
+
+We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend the Operate First Data Science Community Meetups:
+1. Your name
+
+    *a.* as it appears on your Google Account, in case you join through your Google Account; or
+
+    *b.* as you provide in the “Your name” section, in case you join anonymously
+
+2. Email ID or phone number you used to join the event
+3. Timestamp(s) when you joined the event
+4. Timestamp(s) when you left the event
+
+We use **Google Analytics** on the operate-first website. This means that, through Google Analytics, we get information such a your Device's Internet Protocol address (e.g. IP address), browser type, browser version, the pages of our Service that You visit, the time and date of Your visit, the time spent on those pages, unique device identifiers and other diagnostic data. We do **not** get any personal information through google analytics.
```

Comment:
```suggestion
We use **Google Analytics** on the operate-first website. This means that when you visit [operate-first](https://www.operate-first.cloud/) website, through Google Analytics, we get information such as your Device's Internet Protocol address (e.g. IP address), browser type, browser version, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages, unique device identifiers and other diagnostic data. We do **not** get any personal information through google analytics.
```

```
 ..and other diagnostic data. We do **not** get any personal information through google analytics.
```

 The term `other diagnostic data` is making me think what other metrics could be there that is shared? Can we be more specific? wdyt? 

@aakankshaduggal , wonderful doc :clap: . I just added some comments and suggestions. Let me know if it makes sense to you. :) Thanks